ui/pages/settings/security-tab/security-tab.component.js

```diff
@@ -914,18 +911,16 @@ export default class SecurityTab extends PureComponent {
           {this.renderNftDetectionToggle()}
         </div>
 
-        {
-          ///: BEGIN:ONLY_INCLUDE_IF(petnames)
-        }
-        <span className="settings-page__security-tab-sub-header">
-          {this.context.t('settingsSubHeadingSignatures')}
-        </span>
-        <div className="settings-page__content-padded">
-          {this.renderExternalNameSourcesToggle()}
-        </div>
-        {
-          ///: END:ONLY_INCLUDE_IF
-        }
+        {petnamesEnabled ? (
+          <>
+            <span className="settings-page__security-tab-sub-header">
+              {this.context.t('settingsSubHeadingSignatures')}
```

Comment:
I think we should revisit this content here since petnames are not only for signatures but transactions as well. 
![Screenshot 2024-01-18 at 14 06 58](https://github.com/MetaMask/metamask-extension/assets/7644512/e9dc27c3-cebd-4633-b0cb-b09b59cebd2e)

Reply:
Done.